docs/server.md

```diff
@@ -191,24 +193,21 @@ The are four arguments for MMS that facilitate logging of the model serving and
 
 1. **log-level**: optional, log level. By default it is INFO. Possible values are NOTEST, DEBUG, INFO, ERROR and CRITICAL. Check the [Python docs for logging levels](https://docs.python.org/2/library/logging.html#logging-levels) for more information.
 
-1. **metrics-write-to**: various server metrics are gathered and are written to the default log file, but if the `csv` value is passed to this argument, the metrics are recorded every 30 seconds in separate CSV files in a metrics folder in the current directory as follows.
+1. **metrics-write-to**: various server metrics are gathered and are written to the default log file.
 
-      a) **mms_cpu.csv** - CPU load
+  If the `csv` value is passed to this argument, the metrics are recorded every 30 seconds in separate CSV files in a metrics folder in the current directory as follows.
 
+      a) **mms_cpu.csv** - CPU load
       b) **mms_errors.csv** - number of errors
-
-      c) **mms_memory.csv**	- memory utilization
-
+      c) **mms_memory.csv** - memory utilization
       d) **mms_preprocess_latency.csv** - any custom pre-processing latency
-
       e) **mms_disk.csv** - disk utilization
-
       f) **mms_inference_latency.csv** - any inference latency
-
       g) **mms_overall_latency.csv** - collective latency
-
       h) **mms_requests.csv** - number of inference requests
 
+  If the `cloudwatch` value is passed, the above metrics will write to [AWS CloudWatch Service](https://aws.amazon.com/cloudwatch/) every 30 seconds with namespace 'mxnet-model-server'. After [configuring AWS crediential](https://docs.aws.amazon.com/cli/latest/userguide/cli-config-files.html), you will see the metrics are pushed to AWS CloudWatch Service.
```

Comment:
If this is a major feature for the release we might want to dedicate a section to describe what it does, why it might be better than a csv, and how you get it running.

Reply:
it is one of the two main feature adds for this release, so i think we should have some docs for this.

Reply:
Agreed.
This is tracked here: #248